src/Language/LanguageToFontRegistry.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Mpdf\Language;
+
+use Mpdf\MpdfException;
+
+class LanguageToFontRegistry implements LanguageToFontInterface
+{
+	/**
+	 * @var LanguageToFontInterface[]
+	 * @since 8.0
+	 */
+	private $register = [];
+
+	/**
+	 * LanguageToFontRegistry constructor.
+	 *
+	 * @param $classes
+	 *
+	 * @since 8.0
+	 */
+	public function __construct(array $classes)
+	{
+		array_walk($classes, function ($class, $key) {
+			$this->add($key, $class);
+		});
```

Comment:
why not use a simple `foreach` loop?

Reply:
I must have written this right after working in JavaScript where "functional" programming is all the rage.